tests/utils.go

```diff
@@ -2437,3 +2437,13 @@ func RenderTargetcliPod(name, disksPVC string) *k8sv1.Pod {
 		},
 	}
 }
+
+func CheckResultShellCommandOnVmi(vmi *v1.VirtualMachineInstance, cmd, output string, timeout int) {
+	res, err := console.SafeExpectBatchWithResponse(vmi, []expect.Batcher{
+		&expect.BSnd{S: fmt.Sprintf("%s\n", cmd)},
+		&expect.BExp{R: console.PromptExpression},
+	}, timeout)
+	Expect(err).ToNot(HaveOccurred())
+	Expect(res).ToNot(BeEmpty())
```

Comment:
Maybe for consistency, it would be better to use ExpectWithOffset in all cases here:
```suggestion
	ExpectWithOffset(1, err).ToNot(HaveOccurred())
	ExpectWithOffset(1, res).ToNot(BeEmpty())
```

Reply:
Done